sprint9/src/components/items/EditInquiry.tsx

```diff
@@ -0,0 +1,51 @@
+"use Client";
+
+import style from "@/src/styles/items/EditInquiry.module.css";
+import Button from "@/src/components/Button";
+import { useState, useEffect } from "react";
+
+interface EditInquiryProps {
+  comment: string;
+  onCancelClick: () => void;
+  onEditClick: (newComment: string) => void;
+}
+
+export default function EditInquiry({
+  comment,
+  onCancelClick,
+  onEditClick
+}: EditInquiryProps) {
+  const [newComment, setNewComment] = useState(comment);
+  const [isButtonActive, setIsButtonActive] = useState(false);
+
+  const handleSaveClick = () => {
+    onEditClick(newComment);
+  };
+
+  //NOTE: newComment의 타입을 위에서 String으로 지정해주었기 때문에 optional chaning(null,undefined 판명)의 효력이 없다고함.
+  useEffect(() => {
+    setIsButtonActive(newComment.trim().length > 0);
```

Comment:
@annylee0723 이렇게 쓸수도 있어요
```suggestion
    setIsButtonActive(!!newComment.trim());
```